CMakeLists.txt

```diff
@@ -159,6 +160,7 @@ endif()
 if(libOPNMIDI_SHARED)
     add_library(OPNMIDI_shared SHARED ${libOPNMIDI_SOURCES})
     set_target_properties(OPNMIDI_shared PROPERTIES OUTPUT_NAME OPNMIDI)
+    target_include_directories(OPNMIDI_shared PUBLIC ${libOPNMIDI_SOURCE_DIR}/include)
     target_include_directories(OPNMIDI_shared PUBLIC ${SDLMixerX_SOURCE_DIR}/include)
```

Comment:
SDLMixerX_SOURCE_DIR's directory is invalid, copy-pasted from my another project, so, I think, you wanna replace this crap with right lines you have been added above?